camel-core/src/main/java/org/apache/camel/ComponentVerifier.java

```diff
@@ -412,6 +411,12 @@ public String toString() {
         }
     }
 
+    class ExceptionAttributeImpl extends ErrorAttribute implements VerificationError.ExceptionAttribute {
+        private ExceptionAttributeImpl(String name) {
+            super(name);
+        }
+    }
+
```

Comment:
would have to stay in org.apache.camel (which is one of the largest packages btw I ever have seen ;-)